web_src/js/features/repo-home.js

```diff
@@ -6,38 +6,39 @@ import {POST} from '../modules/fetch.js';
 const {appSubUrl} = window.config;
 
 export function initRepoTopicBar() {
-  const $mgrBtn = $('#manage_topic');
-  if (!$mgrBtn.length) return;
-  const $editDiv = $('#topic_edit');
+  const mgrBtn = document.getElementById('manage_topic');
+  if (!mgrBtn) return;
+  const editDiv = document.getElementById('topic_edit');
   const $viewDiv = $('#repo-topics');
```

Comment:
Also `getElementById` this one, just for consistency?

Reply:
https://github.com/go-gitea/gitea/pull/30050/commits/4ec645ef1624dd9626fbda4286b08002259f1da3